src/pages/Troubleshooting.vue

```diff
@@ -84,8 +85,10 @@ export default {
   title:      'Troubleshooting',
   components: { TroubleshootingLineItem, Checkbox },
   data:       () => ({
-    state:    ipcRenderer.sendSync('k8s-state'),
-    settings: defaultSettings,
+    state:           ipcRenderer.sendSync('k8s-state'),
+    settings:        defaultSettings,
+    isDebugging:     defaultSettings.debug,
```

Comment:
Should this be `runInDebugMode(defaultSettings.debug)`? Why not?

Reply:
There's no difference because the `isDebugging` field gets updated using `runInDebugMode(...)` in the `settings-read` handler, but why not. It's more consistent this way.